web/concrete/src/Page/Type/Type.php

```diff
@@ -760,6 +762,9 @@ public static function add($data, $pkg = false)
         $ptID = $db->Insert_ID();
         if ($ptAllowedPageTemplates != 'A') {
             foreach ($templates as $pt) {
+                if(!is_object($pt)){
```

Comment:
Aha! sneaky little bugger. Still getting used to that aspect of PSR-2. Old habits die hard some days. Installed a code analyzer for next time...